nntrainer/models/model_loader.cpp

```diff
@@ -232,13 +244,6 @@ int ModelLoader::loadDatasetConfigIni(dictionary *ini, NeuralNetwork &model) {
     ml_logi("setting labeData is deprecated!, it is essentially noop now!");
   }
 
-  status = model.data_buffer->setBatchSize(model.batch_size);
```

Comment:
is this intentionally removed without any replacement?

Reply:
Yes, it's gotta be done at finalize or before call.

Reply:
cool.